app/models/user.rb

```diff
@@ -963,12 +963,21 @@ def ensure_uniq_email
     raise Exceptions::UnprocessableEntity, 'Email address is already used for other user.'
   end
 
-  def validate_roles
+  def validate_ooo
```

Comment:
Could you please restore the method order to remove the noise from the diff? `validate_roles` should come before `validate_ooo`.